flowauth/Dockerfile

```diff
@@ -24,6 +24,11 @@ ENV FLASK_APP flowauth
 ENV PIPENV_COLORBLIND=1
 WORKDIR /app/static
 COPY --from=builder /build .
-WORKDIR /app
+ARG SOURCE_VERSION=0+unknown
+ENV SOURCE_VERSION=${SOURCE_VERSION}
+ENV SOURCE_TREE=FlowKit-${SOURCE_VERSION}
+WORKDIR /${SOURCE_TREE}/flowauth
 COPY backend .
-RUN pip install --no-cache-dir .
\ No newline at end of file
+RUN python setup.py bdist_wheel && pip install dist/*.whl
+WORKDIR /app
+COPY backend/uwsgi.ini .
```

Comment:
What exactly is the purpose of this `uwsgi.ini` file? Should this be included in the distribution instead (e.g. via `MANIFEST.in`)?

Reply:
It configures uwsgi, which is serving the app in combination with nginx. So is deployment-side rather than distribution side.